pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java

```diff
@@ -58,7 +66,8 @@ public boolean equals(Object obj) {
         InternalConfigurationData other = (InternalConfigurationData) obj;
```

Comment:
We could just use lombok to generate equals and toString methods

Reply:
changed to using lombok